homeassistant/components/binary_sensor/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.entity import Entity
-from homeassistant.const import (STATE_ON, STATE_OFF)
+from homeassistant.const import (ATTR_SENSOR_CLASS, STATE_ON, STATE_OFF)
```

Comment:
I added this thinking that other binary_sensor platforms would want to be allowing you to set the sensor class like I do in nx584 (although I call them zone_types) and in the new template one.

But, I can surely pull it out and only use it in the places it's needed if you prefer.